src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OpprettOppgaveForBarnService.kt

```diff
@@ -0,0 +1,69 @@
+package no.nav.familie.ef.iverksett.oppgave
+
+import no.nav.familie.ef.iverksett.iverksetting.IverksettingRepository
+import no.nav.familie.ef.iverksett.iverksetting.domene.Iverksett
+import no.nav.familie.kontrakter.ef.søknad.Fødselsnummer
+import no.nav.familie.kontrakter.felles.oppgave.Oppgavetype
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+
+@Service
+class OpprettOppgaveForBarnService(private val oppgaveClient: OppgaveClient,
+                                   private val iverksettingRepository: IverksettingRepository) {
+
+    private val secureLogger = LoggerFactory.getLogger("secureLogger")
+
+    fun opprettOppgaverForAlleBarnSomFyller(innenAntallUker: Long) {
```

Comment:
Et risikomoment i funksjonen her: Hvis én feiler underveis vil alle rekjøres på nytt og vi får doble oppgaver for de som gikk igjennom først... 

Bør vi ha en sjekk på at vi ikke allerede har laget denne oppgaven - og klarer vi det?

Ser for meg at  før vi gjør `oppgaveClient.opprettOppgave` kan det sjekkes om det finnes en lignende oppgave for:
- person A, med fagsak B, med type C og beskrivelse D 

? Andre forslag?

Reply:
For å unngå dette kan vi lage en ny task for hver oppgave

Reply:
Er det nok å sjekke om det finnes en oppgave av type "innhent dokumentasjon", og så skippe oppgaveopprettelsen om det er tilfellet ?

Reply:
Satser sjekk på type + lik beskrivelse